coalib/results/Diff.py

```diff
@@ -78,7 +78,7 @@ def from_clang_fixit(cls, fixit, file):
                     fixit.value +
                     oldvalue[endindex:])
         new_file = (file[:fixit.range.start.line-1] +
-                    newvalue.splitlines(True) +
+                    type(file)(newvalue.splitlines(True)) +
                     file[fixit.range.end.line:])
```

Comment:
not sure, we switched to tuples for files because they're immutable, still it might be that a user gives a list I think this is rather safe. If the user passes something obscure this'll most definitely fail